sssd_test_framework/utils/sssd.py

```diff
@@ -783,3 +783,54 @@ def autofs(self) -> None:
         """
         self.sssd.authselect.select("sssd")
         self.sssd.enable_responder("autofs")
+
+    def proxy(self,
+              host: MultihostHost,
```

Comment:
Not needed.

Reply:
This is required to "systemclt restart nsldc"  Or how can we do that ? 

as self.host does not exists